### ISSUE TITLE
Accuracy with given model

### ISSUE THREAD

**Issue by gautamMalu** [aimerykong/deepImageAestheticsAnalysis#2]
Hi,
I was wondering which model is this. Is it the same model which has been referred as Reg+Rank (within & Cross Raters) in the paper. I have tested the given model and these are the results I have got. I have used scipy.stats.spearmanr to calculate spearman rank correlation.
Here are my scripts used in generating these stats https://github.com/gautamMalu/deepImageAestheticsAnalysis

For imgListTestRegression_
score rho: 0.580110099836 at p value: 5.3391175615e-91

For imgListTestNewRegression_
For score rho: 0.654861717641 at p value: 1.6077054187e-123

For imgListValidationRegression_
For score rho: 0.592815662923 at p value: 9.03208424904e-49

For imgListTrainRegression_
For score rho: 0.957560604108 at p value: 0.0

**Comment by aimerykong** [comment]
Hi, Gautam,

Thank you for you interest and sorry for late reply as I'm traveling.

The model released on the website is not the final one trained on AADB; I
guess you are right that it is the Reg+Rank -- I use this model to
train/finetune +Att and +Cont models.

But this might not be the same model used in the paper. Most of my models
were trained in Adobe and they belong to Adobe, meaning that I cannot
release freely.

'imgListTestNewRegression' lists the testset used in our paper, including
images cleansed from "imgListTestRegression".

Hope this helps.

Regards,
Shu

**Comment by gautamMalu** [comment]
Thanks for the clarification. Good luck for your ECCV presentation.
